annotationProcessor/gradle.lockfile

```diff
@@ -1,6 +1,7 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
+args4j:args4j:2.33=productionRuntimeClasspath,runtimeClasspath,testFixturesRuntimeClasspath,testRuntimeClasspath
```

Comment:
Changes due to adding indexer as a dependency to service. 
(annotationProcessor already depends on service)